packages/shared/src/components/HorizontalScroll/useHorizontalScrollHeader.tsx

```diff
@@ -71,37 +67,21 @@ export const useHorizontalScrollHeader = <
     }
   }, [numCards, scrollableElementWidth, onScroll]);
 
-  const Header = useMemo(
-    () =>
-      // eslint-disable-next-line react/display-name
-      (props: HeaderProps = {}) =>
-        (
-          <HorizontalScrollHeader
-            {...props}
-            title={title}
```

Comment:
We always override the title here, so no matter what we pass through the props, it will get overridden. That means, we have no more purpose for this component since what it only accepts is the title.